packages/components/src/tabs/src/components/osds-tabs/core/controller.ts

```diff
@@ -1,52 +1,80 @@
+import type { OsdsTabBarItem } from '../../osds-tab-bar-item/osds-tab-bar-item';
+import type { OsdsTabsPanel } from '../../osds-tab-panel/osds-tab-panel';
 import type { OsdsTabs } from '../osds-tabs';
 
-import { OsdsTabBarItem } from '../../osds-tab-bar-item/osds-tab-bar-item';
-import { OsdsTabsPanel } from '../../osds-tab-panel/osds-tab-panel';
-
 /**
  * common controller logic for text component used by the different implementations.
  * it contains all the glue between framework implementation and the third party service.
  */
 class OdsTabsController {
   protected component: OsdsTabs;
 
+  private _tabItems: (Array<OsdsTabBarItem & HTMLElement>) = [];
```

Comment:
as the getter and setter does nothing special, you can set this protected or public directly

Reply:
changed